packages/ibm-products/src/components/UserAvatar/_storybook-styles.scss

```diff
@@ -12,11 +12,13 @@
 
 // TODO: add any additional styles used by UserAvatar.stories.js
 .theme-section {
+  /* stylelint-disable-next-line carbon/layout-use */
```

Comment:
```suggestion
```

Assuming these _should_ work when namespaced, but if not, we may need to keep this in place for now.